src/components/RegionalUrls.js

```diff
@@ -1,17 +1,25 @@
 import React, { useState, useEffect } from "react";
+import CodeBlock from "@theme/CodeBlock";
 
 const DOMAINS = {
   US: {
     langsmith: "smith.langchain.com",
     api: "api.smith.langchain.com",
+    hub: "api.hub.langchain.com",
```

Comment:
can we deprecate this? api should be same url as hub now. In fact we should **ony** need to specify the api url(hub sdk is bundled into main langsmith sdk now)

Reply:
since it was bundled recently, ok to include for now and remove in a couple months?
@madams0013 thoughts on deprecation strategy?

Reply:
I think in other places in the docs we specify "only use this if using the legacy langchainhub sdk)

Reply:
updated the wording and moved the note to make this clearer

Reply:
also can we have it poiint to the main api url? instead of the .hub url?

Reply:
we don't need the url to be a separate domain anymore either, we just keep for legacy reasons

Reply:
updated